src/main/kotlin/org/rust/lang/utils/RsDiagnostic.kt

```diff
@@ -512,7 +512,7 @@ sealed class RsDiagnostic(
         }
     }
 
-    class TooFewParamsError(
+    class IncorrectArgumentCountErrorVariadic(
```

Comment:
Maybe it's better to merge `IncorrectArgumentCountErrorVariadic` with `IncorrectArgumentCountError`? Something like this:
```kotlin
class IncorrectArgumentCountError(
    element: PsiElement,
    private val expectedCount: Int,
    private val realCount: Int,
    private val variadic: Boolean
) : RsDiagnostic(element) {
    override fun prepare() = PreparedAnnotation(
        ERROR,
        if (variadic) E0060 else E0061,
        errorText()
    )

    private fun errorText(): String {
        return "This function takes${if (variadic) " at least" else ""}" +
            " $expectedCount ${pluralise(expectedCount, "parameter", "parameters")}" +
            " but $realCount ${pluralise(realCount, "parameter", "parameters")}" +
            " ${pluralise(realCount, "was", "were")} supplied"
    }
}
```